### DISCUSSION TITLE
JSON error on system side

### DISCUSSION THREAD

**Discussion by dukeplay** [rern/rAudio#643] [Fixed Issues]
Hello rern, thank you for the great player and for your valuable work. Since the last update 20211121 I got this error on the system side: Errors: Unexpected token] in JSON at position 1808. Maybe you can look over it again. Thanks very much!

**Comment by ceejayemm** [comment]
@rern 

Same here but at position 1812.

Chris

**Comment by rern** [comment]
Let me see the error message.

**Comment by dukeplay** [comment]
Errors: Unexpected token ] in JSON at position 1809
{

  "page"             : "system"
, "cpuload"          : "0.00 0.01 0.00"
, "cputemp"          : 53.0
, "startup"          : "4s (kernel) + 7s (userspace) "
, "throttled"        : "0x0"
, "time"             : "10:17:58 2021-11-21"
, "timezone"         : "Europe/Berlin"
, "uptime"           : "22m"
, "uptimesince"      : "2021-11-21 09:55"
, "audioaplayname"   : "iqaudio-dacplus"
, "audiooutput"      : "IQaudIO DAC+"
, "bluetooth"        :false
, "bluetoothactive"  :false
, "bluetoothconf"    : [ true, false ]
, "firmware"         : "20211004-1"
, "hostapd"          :false
, "hostname"         : "rAudio"
, "kernel"           : "5.10.74-1-raspberrypi4-ARCH aarch64"
, "lcd"              :false
, "lcdchar"          : false
, "lcdcharaddr"      : "0x27 0x3F"
, "lcdcharconf"      : [ 16,"A00","i2c","0x27","PCF8574",15,18,16,21,22,23,24,false ]
, "list"             : [ {"icon":"microsd","mountpoint":"/","mounted":true,"source":"/dev/mmcblk0p2","size":"3.4GB/59GB"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/NAS","mounted":true,"source":"//192.168.20.26/Musik/music","size":"1.1TB/1.8TB"} ]
, "lcdmodel"         : "tft35a"
, "mpdoled"          : true
, "mpdoledconf"      : [ "6", 400000 ]
, "online"           : true
, "ntp"              : "pool.ntp.org"
, "powerbutton"      : true
, "powerbuttonconf"  : [ 16,8,5 ]
, "relays"           : false
, "rpimodel"         : "Raspberry Pi 4 Model B Rev 1.1"
, "soc"              : "BCM2711"
, "soccpu"           : "4"
, "socram"           : "3.7GiB"
, "socspeed"         : "1500"
, "soundprofile"     : true
, "soundprofileconf" : [ 1500000,0,1500,1000 ]
, "version"          : "1"
, "versionui"        : 20211121
, "vuled"            : false
, "vuledconf"        : [ 14,15,18,23,24,25,8 ]
, "wlan"             : true
, "wlanconf"         : [
  "DE"
, 
▮]
, "wlanconnected"    : true
}

**Comment by dukeplay** [comment]
It doesn't belong under this topic now, but the Chromium browser on my DSI display brings the following error message: "Page Unresponsive. You can wait for it to become responsive or exit the page."

**Reply by dukeplay** [reply]
If you click on Exit Page and Reload, the page works. After the restart, errors on the display myself.

**Comment by ceejayemm** [comment]
Mine is the same as @dukeplay except for the 'wlanconf' parameter which is 'GB'

Chris

**Comment by ceejayemm** [comment]
Fixed for me with 20211122

Thanks

Chris

**Comment by dukeplay** [comment]
Thank you for the quick update!
